core/src/main/scala/org/scalatra/StableResult.scala

```diff
@@ -0,0 +1,20 @@
+package org.scalatra
+
+import javax.servlet.ServletContext
+import javax.servlet.http.{ HttpServletResponse, HttpServletRequest }
+
+// Provides a stable request/response to an action.
+// Each action is wrapped in a StableResult during compilation.
+abstract class StableResult(
```

Comment:
Is it possible to make this package-protected?  It's an implementation detail, which we might figure out how to improve again later.  I wouldn't want to encourage anybody to use this directly.